gunicorn/http/message.py

```diff
@@ -324,6 +330,47 @@ def parse_proxy_protocol(self, line):
             "proxy_port": d_port
         }
 
+    def parse_proxy_protocol_v2(self, unreader, buffer):
+        data = self.read_bytes(unreader, buffer, 4)
+        ver_cmd, fam, length = struct.unpack("!BBH", data)
+        if ver_cmd & 0xF0 != 0x20:
+            raise InvalidProxyHeader("invalid version %r" % data)
+        if ver_cmd & 0xF not in {0x00, 0x01}:
+            raise InvalidProxyHeader("unsupported command %r" % data)
+
+        body = self.read_bytes(unreader, buffer, length)
+        if ver_cmd & 0xF == 0x00:
+            return  # LOCAL command, do not change source addr
+
+        if fam == 0x11:  # TCPv4
```

Comment:
How about to make constants with meaningful names for `0x11` and so on instead comments? Or [enum.IntFlag](https://docs.python.org/3/library/enum.html#enum.IntFlag)?

Reply:
Sounds like a plan, will see what I can come up with.